tests/test_ggsdk.py

```diff
@@ -1,5 +1,5 @@
-# Test ideas
-
-# handling expected senarios of what boot-nlp will give back
-
-# can test sdk calls by creating a obj we know the request/response should look like, then running the code to create those request/response and checking them against each other
\ No newline at end of file
+"""Sdk test ideas. Handling expected scenarios of what boot-nlp will give
```

Comment:
This is pretty minor, but switching these to a single docstring doesn't really make sense here, as this docstring is now defining the use case for this python module `test_ggsdk`

meaning when I run something like:
```python
from tests import test_ggsdk
help(test_ggsdk) 
```
I get:
```text
Help on module tests.test_ggsdk in tests:

NAME
    tests.test_ggsdk

DESCRIPTION
    Sdk test ideas. Handling expected scenarios of what boot-nlp will give
    back. Can test sdk calls by creating an obj we know the request/response
    should look like, then running the code to create those request/response and
    checking them against each other.

FILE
    ~/git/graphgrid-sdk-python/tests/test_ggsdk.py
```

Which doesn't really make sense, as these are thoughts/code comments, rather than instructions for how to use this module. So it'd probably make sense to capture this as some sort of todo with an asana task link (or probably just the task id) and then these description get moved there.